GoogleForCreators/web-stories-wp
# Dashboard: Applied design updates to DIY template

## Summary

This PR applies the new design updates to the DIY template.

## User-facing changes

Layout and text sizes have been update on the DIY template based off the recommendations found in this [figma design doc](https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=766%3A61).

## Testing Instructions

1.) Go to "Explore Templates".
2.) Hover over the DIY template and click on "View".
3.) Take a look at each template page in the detail view for DIY.  Make sure images are showing up and the layout appears as shown in the figma updates.

---

Fixes #2470 

## Screenshots from Editor

<img src="https://user-images.githubusercontent.com/40646372/85317225-e3d6d480-b472-11ea-9265-200f2aabd674.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85317237-eb967900-b472-11ea-9f37-fb37ec277fa0.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85317255-f3eeb400-b472-11ea-8efd-1122ae09cbb6.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/85317265-fb15c200-b472-11ea-9e8c-e297e818786c.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85317278-01a43980-b473-11ea-85ee-04459e566968.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85317298-08cb4780-b473-11ea-99ae-5b95ce3a7699.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/85317318-11238280-b473-11ea-962d-0934d6151bef.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85317859-f1408e80-b473-11ea-9ccf-67e9c00d3985.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85317363-200a3500-b473-11ea-8f1e-01c0aba375c0.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/85317384-27c9d980-b473-11ea-8557-36573adac2c9.png" height="200">
